test/sanity/app-getproxyforurl/index.html

```diff
@@ -15,6 +15,14 @@
 	} else {
 		document.write('<h1 id="result">failure: expect "' + expect + '" but get ' + proxy +'</h1>');
 	}
+        nw.App.setProxyConfig('dummy:8080', 'http://foopy/proxy.pac');
+        var proxy = nw.App.getProxyForURL('http://www.example.com/');
+        var expect = 'DIRECT';
+        if (proxy === expect) {
+                document.write('<h1 id="result">success</h1>');
+        } else {
+                document.write('<h1 id="result">failure: expect "' + expect + '" but get ' + proxy +'</h1>');
+        }
 	</script>
```

Comment:
Use the following command to run test cases?
python test/test.py -t 80 auto
python test/test.py -t 80 remoting